.docker/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM splitbrain/phpfarm:jessie
+FROM php:7.4-apache
```

Comment:
Is there a specific reason why you chose PHP 7.4 over a more current version? PHP 7.4 is EOL since end of 2022.

Reply:
Just saw that 7.4 was used until now, so updating the PHP version is probably out of scope for this PR.